nslsii/motors/delta_tau.py

```diff
@@ -0,0 +1,45 @@
+"""
+Ophyd devices and signals for Delta Tau devices, such as Programmable Multi-Axis Controller (PMAC-PC)
+"""
+
+import logging
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal, EpicsSignalRO, Kind
+
+
+class PMACStatus(EpicsSignalRO):
+    """
+    A signal to read the status of the PMAC-PC controller by checking the bit 11 of the status register.
+    """
+
+    def get(self):
+        return int(super().get() >> 11 & 1 == 0)
+
+
+class PMACKillSwitch(Device):
+    """
+    A device to kill the PMAC-PC controller.
+    In the named version of the EPICS record (e.g., XF:23ID1-ES{Dif-Ax:Del}), the status is expected at Sts:4-Sts.
+    Whereas in the axis numbered version (e.g., XF:23ID1-CT{MC:12-Ax:1}), the status is expected at Sts:1-Sts.
+    """
+
+    kill = Cpt(EpicsSignal, "Cmd:Kill-Cmd.PROC", kind=Kind.omitted)
+    status = Cpt(PMACStatus, "Sts:4-Sts", kind=Kind.normal)
+
```

Comment:
The naming convention here afaik is actually based on the axis number. So for axis one, it would be Sts:1-Sts. For axis 2 it would be Sts:2-Sts and so on. So probably axis number should be an argument passed in at `__init__` and the component should be generated based on that.